measurements/api/measurements.py

```diff
@@ -321,6 +321,118 @@ def _merge_results(tmpresults):
     return tuple(resultsmap.values())
 
 
+def get_measurement_meta(report_id=None, input=None):
+    """Get metadata on one measurement by measurement_id + input
+    """
```

Comment:
You probably want to ensure that `report_id` is not NULL, either in here or in the swagger spec.

Reply:
Done